README.Rmd

```diff
@@ -125,6 +125,35 @@ mins_to_hours <- function(data) {
 ark(db, dir, lines = 50000, tables = "flights", callback = mins_to_hours)
 ```
 
+## ark() in parallel
+
+There are two strategies for using `ark` in parallel. One is to loop over the tables, re-using the ark function per table in parallel. The other, introduced in 0.0.15, is to use the "window-parallel" method. This is particularly useful if your tables are very large and can speed up the process significantly. 
+
+```{r, eval = FALSE}
+# Strategy 1
+library(arkdb)
+library(future.apply)
+
+plan(multisession)
+
+future_lapply(vector_of_tables, function(x) ark(db, dir, lines, tables = x))
+
+# Strategy 2
+library(arkdb)
+library(future.apply)
+
+plan(multisession)
+
+ark(db, dir, lines = 50000, tables = "flights", method = "window-parallel")
```

Comment:
Love the examples here. Just to be clear, does `method = "window-parallel"` or  imply we are writing to (chunked) parquet files?

Reply:
Yes, this is a good point. I didn't setup the other methods to accept a filename argument. SHould probably setup a stop condition here. I'll update.

Reply:
Resolving this. Added a stop condition + test for stop.